FirebaseMessaging/Tests/IntegrationTests/FIRMessagingTokenRefreshTests.swift

```diff
@@ -97,6 +95,30 @@
       wait(for: [expectation, notificationExpectation], timeout: 5)
     }
 
+    func testDeleteDataWithTokenRefreshDelegatesAndNotifications() {
+      let expectation = self.expectation(description: "delegate method and notification are called")
+      assertDefaultToken()
+
+      let notificationExpectation = self.expectation(forNotification: NSNotification.Name
+        .MessagingRegistrationTokenRefreshed,
+        object: nil,
+        handler: nil)
+
+      let testDelegate = fakeAppDelegate()
+      messaging?.delegate = testDelegate
+      testDelegate.delegateIsCalled = false
+
+      guard let messaging = self.messaging else {
+        return
+      }
```

Comment:
Based on previous refactor you did, can this now be?:
```suggestion
      messaging.delegate = testDelegate
      testDelegate.delegateIsCalled = false
```

Reply:
Done!